src/lib/utils/meta_tables/meta_system_utilization_table.cpp

```diff
@@ -260,6 +260,23 @@ MetaSystemUtilizationTable::ProcessMemoryUsage MetaSystemUtilizationTable::_get_
  * allocated memory as free'd memory is not immediately returned to the system (either due to internal page
  * fragmentation or because jemalloc keeps empty pages for future use). In rare cases, it can also be higher than
  * the amount of virtual memory if memory has been allocated but not committed yet.
+ *
+ * jemalloc's memory allocation uses different size classes, each with a different size of bytes to allocate. If a
+ * data structure's allocator requests a specific amount of bytes that exceeds a certain size class by only one byte,
+ * the next larger size class is used and the full amount of bytes of this class is allocated. The spacing between
+ * size classes doubles every 4th class. Consequently, the larger the size class that the requested amount of memory
```

Comment:
Regarding the doubling every 4th class, would this result in `20, 24, 28, 32, 40, 48, 56, 64, 80`?

Reply:
Correct